src/spikeinterface/core/tests/test_analyzer_extension_core.py

```diff
@@ -111,7 +111,7 @@ def test_ComputeWaveforms(format, sparse):
 def test_ComputeTemplates(format, sparse):
     sorting_analyzer = get_sorting_analyzer(format=format, sparse=sparse)
 
-    sorting_analyzer.compute("random_spikes", max_spikes_per_unit=20, seed=2205)
+    sorting_analyzer.compute("random_spikes", max_spikes_per_unit=500, seed=2205)
```

Comment:
This slow down a lot tests. I prefer a small number!!

Reply:
It forsnt